### ISSUE TITLE
LogicException in Command.php line 65: The command defined in "App\Http\Controllers\mycontroller" cannot have an empty name.

### ISSUE THREAD

**Issue by ImmanuelJL** [nunomaduro/laravel-desktop-notifier#6]
Hello, i try to install and get this error:

LogicException in Command.php line 65:
The command defined in "App\Http\Controllers\mycontroller" cannot have an empty name.

Here my full code:

namespace App\Http\Controllers;

use Illuminate\Console\Command;
use NunoMaduro\LaravelDesktopNotifier\Notifications;

class mycontroller extends Command
{
    use Notifications;

    public function handle()
    {
        $this->notify('Hello Web Artisan', 'Love beautiful code? We do too!');
    }
}

Did i do something wrong?
Any way thanks for this good package

**Comment by nunomaduro** [comment]
Hey 👋 

Thanks for notice that potencial issue. I will be able to verify it only after my holidays on 20/08!

**Comment by ImmanuelJL** [comment]
Okayyy bro...

**Comment by nunomaduro** [comment]
Hey @ImmanuelJL,

On the code you provided you forgot to include the properties signature/name and description.

Example:

```
class Zonda extends Command
{
    /**
     * The name and signature of the console command.
     *
     * @var string
     */
    protected $signature = 'zonda';

    /**
     * The console command description.
     *
     * @var string
     */
    protected $description = 'Command description';

```

**Comment by nunomaduro** [comment]
Try it and tell me if it works.

**Comment by nunomaduro** [comment]
@ImmanuelJL I am closing this for lack of response.

**Comment by bacancy-keyur** [comment]
The "Name" property should be written along with signature like in the extension to the @nunomaduro's sample,

    class MergeTable extends Command
    {

        /**
         * The console command name.
         *
         * @var string
         */
        protected $name = "merge-table";

        /**
         * The name and signature of the console command.
         *
         * @var string
         */
        protected $signature = "merge-table";

        /**
         * The console command description.
         *
         * @var string
         */
        protected $description = "Command description";

This solved my problem in Laravel 5.0, Not sure about the above versions.
